client/src/pages/SearchResultPage.vue

```diff
@@ -55,12 +55,12 @@
 					{{ section.header }}
 				</GridItem>
 
-				<SearchResultCard
-					v-for="record in section.records"
-					:key="record.type"
-					data-test="search-results-cards"
-					:data-source="searchResult[record.type]"
-				/>
+				<ErrorBoundary v-for="record in section.records" :key="record.type">
```

Comment:
Note: `ErrorBoundary` can be added anywhere. For example, in this `v-for`, we can stop the error from bubbling all the way up the component tree and only render the error message in the single card that failed, so the user can still interact with the rest of the UI.